src/layouts/Homepage/index.jsx

```diff
@@ -34,6 +34,7 @@ export default class Homepage extends DefaultTemplate {
       <Html>
         <Head
           title={this.props.file.title}
+          description={i18n.description}
```

Comment:
My solution gives the opportunity to add descriptions for other layouts/pages, no?